app/views/assessor_interface/assessment_sections/show.html.erb

```diff
@@ -46,11 +46,20 @@
   <% end %>
 <% end %>
 
-<% if @assessment_section_view_object.show_online_checker? %>
-  <h2 class="govuk-heading-s">Online checker</h2>
-  <p class="govuk-body">This authority has an online checker for validating the supplied reference number:</p>
-  <p class="govuk-body"><%= govuk_link_to @assessment_section_view_object.online_checker_url, @assessment_section_view_object.online_checker_url, target: "_blank" %></p>
-<% end %>
+<%= govuk_accordion do |accordion| %>
+  <% if @assessment_section_view_object.show_online_checker? %>
+    <% accordion.section(heading_text: "Online checker") do%>
+      <p class="govuk-body">This authority has an online checker for validating the supplied reference number:</p>
+      <p class="govuk-body"><%= govuk_link_to @assessment_section_view_object.online_checker_url, @assessment_section_view_object.online_checker_url, target: "_blank" %></p>
+    <% end %>
+  <% end %>
+
+  <%if @assessment_section_view_object.show_registration_number_summary%>
```

Comment:
This method doesn't look to check that we're rendering the professional recognition assessment section, it only checks whether a registration number has been set:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/ded278a4baace9d0e94a1ed0a6ca051b9a4f86d2/app/view_objects/assessor_interface/assessment_section_view_object.rb#L36-L38

However, I think we want this accordion to show whenever a user is reviewing professional recognition information, regardless of whether it's a written statement or a registration number.

Reply:
Done, I think 👍